tests/integration_tests/security/test_spectre_meltdown.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# N.B.: Although this repository is released under the Apache-2.0, part of its test requires a
+# script from the third party "Spectre & Meltdown Checker" project. This script is under the
+# GPL-3.0-only license.
+"""Tests spectre / meltdown mitigation."""
+
+import pytest
+import requests
+
+from framework import utils
+from host_tools.network import SSHConnection
+
+
+CHECKER_URL = "https://meltdown.ovh"
+CHECKER_FILENAME = "spectre-meltdown-checker.sh"
+
+
+@pytest.fixture(name="spectre_metldown_checker")
+def download_spectre_meltdown_checker(tmp_path):
+    """Download spectre / meltdown checker script."""
+    resp = requests.get(CHECKER_URL, timeout=5)
+    assert (
+        resp.ok
+    ), f"Failed to download the checker script. Status code: {resp.status_code}"
+
+    path = tmp_path / CHECKER_FILENAME
+    with open(path, "wb") as file:
+        file.write(resp.content)
+
+    return path
+
+
+def test_spectre_meltdown_checker_on_host(spectre_metldown_checker):
+    """
+    Run the spectre / meltdown checker script on host.
+
+    @type: security
+    """
+    utils.run_cmd(f"sh {spectre_metldown_checker} --paranoid --explain")
```

Comment:
Is this printing the script's output?
It would be nice to programmatically inspect if the checks pass and fail the test in case they don't.

Reply:
Yep. `utils.run_cmd()` will print stdout and stderr when the check fail.